deploy/warehouse/target/data.sql

```diff
@@ -48,4 +112,15 @@ insert into warehouse.target (identifier, organism_id, control)
         set organism_id = EXCLUDED.organism_id
 ;
 
+
+insert into warehouse.target (identifier, control)
+    values
+        ('Hs04930436_g1',   't'),
+        ('Ac00010014_a1',   't')
+
+    on conflict (identifier) do update
+        set identifier  = coalesce(target.identifier, '')   || EXCLUDED.identifier,
+            control     = EXCLUDED.control
```

Comment:
does this need a `coalesce()`? My guess is no since `f` is the default value.

Reply:
There is no need for a coalesce here. We want to set control to the value we hardcode here regardless of what its previous value was.